docs/usage.md

```diff
@@ -3,132 +3,488 @@
+
+### Review Linked Records
+
+- Verify the accuracy and relevance of the linked records. Ensure that the connections provide a clear understanding of the relationships and contribute to effective analysis.
+
+### Integrate Linked Records into Workflow
+
+- Utilize the linked records to enhance your workflow. The connections between records can provide valuable insights and help streamline data analysis and response actions.
+
+Linking records is crucial for establishing connections between related pieces of information within FortiSOAR.
+
+>[!TIP]
+>When linking records, ensure that the connections are relevant and meaningful. Properly linking records can provide a more comprehensive view and improve the effectiveness of your analysis and response.
+
+## Create Record
+
+Creating records within the FortiAI assistant involves generating new entries for specified types, pages, or modules. This functionality allows users to add new data to the system, such as creating a new alert or incident record, to manage and track information effectively.
+
+>[!NOTE]
+>Ensure that you provide all the relevant details for records to be accurate and complete. Proper record creation supports effective data management and enhances the overall workflow.
+
+### Use Case Example: Creating a New Incident
+
+A common use case is creating a new incident record when a new threat is detected. For example, if an alert indicates a potential security breach, a new incident record can be created to track and manage the response.
+
+### Creating Records Using FortiAI Assistant
+
+1. **Initiate Record Creation**:
+   - Issue a command to create a record for the specified type, page, or module. For example:
+
+    >*Create a new incident record in the Incidents module*
+
+   - The assistant prompts you to provide the necessary details for the new record.
+
+2. **Specify Record Details**:
+   - Provide the required information for the record being created. For example:
+
+    >*Create a new alert with the following details: `[Details of the alert]`*
+
+   - The assistant generates the new record based on the provided details. If any mandatory details are missed, the bot requests for the missing information.
+
+### Review Created Records
+
+- Verify the accuracy and completeness of the created records. Ensure that all necessary information is included and that the record meets the required specifications.
+
+### Integrate Created Records into Workflow
+
+- Utilize the newly created records within your workflow. The records can be used for tracking, analysis, and response actions, contributing to an efficient data management process.
+
+Creating records is a fundamental task for managing and tracking data within FortiSOAR. By following the steps outlined and leveraging the FortiAI assistant’s capabilities, users can effectively create and manage records to support their workflow and data management needs.
+
+>[!NOTE]
```

Comment:
Add this text in note for create and update record section.
On MSSP setup, user will not be able to create record with a prompt containing tenant name like "Create a new alert for tenant mytenant"

Reply:
Done